lmcache/v1/storage_backend/__init__.py

```diff
@@ -198,17 +198,20 @@ def CreateStorageBackends(
         backend_name = str(remote_backend)
         storage_backends[backend_name] = remote_backend
 
-    # Create dynamic backends from configuration
-    create_dynamic_backends(
-        config,
-        metadata,
-        loop,
-        memory_allocator,
-        local_cpu_backend,
-        dst_device,
-        lookup_server,
-        storage_backends,
-    )
+    if config.enable_nixl and not config.local_cpu:
+        pass
+    else:
+        # Create dynamic backends from configuration
+        create_dynamic_backends(
+            config,
+            metadata,
+            loop,
+            memory_allocator,
+            local_cpu_backend,
+            dst_device,
+            lookup_server,
+            storage_backends,
+        )
```

Comment:
Using a placeholder like `pass` is ok in test code but not here. I think the code below is cleaner and is easier to understand the logic.

```suggestion
    if not config.enable_nixl or config.local_cpu:
        # Create dynamic backends from configuration
        create_dynamic_backends(
            config,
            metadata,
            loop,
            memory_allocator,
            local_cpu_backend,
            dst_device,
            lookup_server,
            storage_backends,
        )
```

Reply:
@Shaoting-Feng If you are ok with the above suggestion, do you mind also changing it in https://github.com/LMCache/LMCache/blob/dev/lmcache/v1/storage_backend/__init__.py#L142-L152?

Reply:
Thanks! It’s nice to learn this.